README.md

```diff
@@ -49,6 +49,7 @@ Given the following network configuration:
 		"type": "bond",
 		"mode": "active-backup",
 		"miimon": "100",
+        "failOverMac": "1",
```

Comment:
remove double quotes for value 1, otherwise, it complains bondingConfig.failOverMac type error.

Reply:
Good catch - thanks for that.